missing gcc configure "--enable-plugin" package for windows

hi
i'm using windows Mingw and couldn't find a copmlete build of Mingw with latest gcc that was gcc configure with "--enable-plugin"
can you please build and upload a new build with gcc configured with "----enable-plugin" ? it should generate a new folder that is now missing in the latest build
the new folder should be generate in "mingw32\lib\gcc\i686-w64-mingw32\8.3.1" as a "plugin" folder with some libs and includes so it would be possible to compile plugins also in windows build..
it was published here that this is now supported :
https://www.phoronix.com/scan.php?page=news_item&px=GCC-Plugins-Windows-MinGW

> Please can you post the error message with the build configuration you used

build configuration is from PKGBUILD+`--enable-plugin`
error messages:
```
  g++ -std=c++11 -no-pie   -O2 -D__USE_MINGW_ACCESS -DIN_GCC     -fno-exceptions -fno-rtti -fasynchronous-unwind-tables -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-format -Wmissing-format-attribute -Wconditionally-supported -Woverloaded-virtual -pedantic -Wno-long-long -Wno-variadic-macros -Wno-overlength-strings   -DHAVE_CONFIG_H -static-libstdc++ -Wl,--no-seh -Wl,--large-address-aware -Wl,--large-address-aware -Wl,--stack,12582912 -Wl,--undefined=HOST_EXTRA_OBJS_SYMBOL -o cc1plus.exe \
        cp/cp-lang.o c-family/stub-objc.o cp/call.o cp/class.o cp/constexpr.o cp/constraint.o cp/coroutines.o cp/cp-gimplify.o cp/cp-objcp-common.o cp/cp-ubsan.o cp/cvt.o cp/contracts.o cp/cxx-pretty-print.o cp/decl.o cp/decl2.o cp/dump.o cp/error.o cp/except.o cp/expr.o cp/friend.o cp/init.o cp/lambda.o cp/lex.o cp/logic.o cp/mangle.o cp/mapper-client.o cp/mapper-resolver.o cp/method.o cp/module.o cp/name-lookup.o cp/optimize.o cp/parser.o cp/pt.o cp/ptree.o cp/rtti.o cp/search.o cp/semantics.o cp/tree.o cp/typeck.o cp/typeck2.o cp/vtable-class-hierarchy.o attribs.o c-family/c-common.o c-family/c-cppbuiltin.o c-family/c-dump.o c-family/c-format.o c-family/c-gimplify.o c-family/c-indentation.o c-family/c-lex.o c-family/c-omp.o c-family/c-opts.o c-family/c-pch.o c-family/c-ppoutput.o c-family/c-pragma.o c-family/c-pretty-print.o c-family/c-semantics.o c-family/c-ada-spec.o c-family/c-ubsan.o c-family/known-headers.o c-family/c-attribs.o c-family/c-warn.o c-family/c-spellcheck.o i386-c.o winnt-c.o winnt-cxx.o msformat-c.o cc1plus-checksum.o libbackend.a main.o libcommon-target.a libcommon.a ../libcpp/libcpp.a ../libdecnumber/libdecnumber.a ../libcody/libcody.a  \
  	libcommon.a ../libcpp/libcpp.a  /mingw32/lib/libiconv.a ../libbacktrace/.libs/libbacktrace.a ../libiberty/libiberty.a ../libdecnumber/libdecnumber.a  -L/mingw32/lib -lisl -L/mingw32/lib -L/mingw32/lib -L/mingw32/lib -lmpc -lmpfr -lgmp -Wl,--export-all-symbols -Wl,--out-implib=cc1plus.exe.a  -lz -lzstd 
  D:/M/msys64/mingw32/bin/../lib/gcc/i686-w64-mingw32/13.2.0/../../../../i686-w64-mingw32/bin/ld.exe: cannot export HOST_EXTRA_OBJS_SYMBOL: symbol not found
  D:/M/msys64/mingw32/bin/../lib/gcc/i686-w64-mingw32/13.2.0/../../../../i686-w64-mingw32/bin/ld.exe: cannot export HOST_EXTRA_OBJS_SYMBOL: symbol not found
  D:/M/msys64/mingw32/bin/../lib/gcc/i686-w64-mingw32/13.2.0/../../../../i686-w64-mingw32/bin/ld.exe: cannot export HOST_EXTRA_OBJS_SYMBOL: symbol not found
  collect2.exe: error: ld returned 1 exit status
  make[3]: *** [../../gcc-13.2.0/gcc/lto/Make-lang.in:96: lto1.exe] Error 1
  make[3]: *** Waiting for unfinished jobs....
  D:/M/msys64/mingw32/bin/../lib/gcc/i686-w64-mingw32/13.2.0/../../../../i686-w64-mingw32/bin/ld.exe: cannot export HOST_EXTRA_OBJS_SYMBOL: symbol not found
  collect2.exe: error: ld returned 1 exit status
  make[3]: *** [../../gcc-13.2.0/gcc/c/Make-lang.in:87: cc1.exe] Error 1
  collect2.exe: error: ld returned 1 exit status
  make[3]: *** [../../gcc-13.2.0/gcc/cp/Make-lang.in:145: cc1plus.exe] Error 1
  collect2.exe: error: ld returned 1 exit status
  make[3]: *** [../../gcc-13.2.0/gcc/ada/gcc-interface/Make-lang.in:758: gnat1.exe] Error 1
  rm gcc.pod
  make[2]: *** [Makefile:4981: all-stage1-gcc] Error 2
  make[1]: *** [Makefile:25599: stage1-bubble] Error 2
  make: *** [Makefile:26538: profiledbootstrap] Error 2
```

--- a/mingw-w64-gcc/PKGBUILD
+++ b/mingw-w64-gcc/PKGBUILD
@@ -30,7 +30,7 @@ pkgver=13.2.0
 #_sourcedir=${_realname}-${_majorver}-${_snapshot}
 _sourcedir=${_realname}-${pkgver}
 _libdir=lib/gcc/${CHOST}/${pkgver%%+*}
-pkgrel=5
+pkgrel=6
 pkgdesc="GCC for the MinGW-w64"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64')
@@ -193,27 +193,24 @@ prepare() {
 build() {
   mkdir -p "${srcdir}"/build-${MSYSTEM} && cd "${srcdir}"/build-${MSYSTEM}
 
-  declare -a extra_config
+  declare -a _extra_config
   if check_option "debug" "n"; then
-    extra_config+=("--disable-libstdcxx-debug")
+    _extra_config+=("--disable-libstdcxx-debug")
   else
-    extra_config+=("--enable-libstdcxx-debug")
+    _extra_config+=("--enable-libstdcxx-debug")
   fi
 
-  case "${CARCH}" in
-    i686)
-      extra_config+=(
-        "--disable-sjlj-exceptions"
-        "--with-dwarf2"
-      )
-      LDFLAGS+=" -Wl,--large-address-aware"
-      local _arch=pentium4
-    ;;
-
-    x86_64)
-      local _arch=nocona
-    ;;
-  esac
+  if [[ "${CARCH}" == "i686" ]]; then
+    _extra_config+=(
+      "--disable-sjlj-exceptions"
+      "--with-dwarf2"
+    )
+    LDFLAGS+=" -Wl,--large-address-aware"
+    local _arch=pentium4
+  else
+    _extra_config+=("--enable-plugin")
+    local _arch=nocona
+  fi
 
   local _languages="c,lto,c++,fortran"
   if [ "$_enable_ada" == "yes" ]; then
@@ -238,7 +235,7 @@ build() {
   # linked dynamically like gmp/zlib/zstd and those pull in libgcc we can't
   # allow libgcc to be linked statically. The default is "-static-libstdc++
   # -static-libgcc" for both, so we drop "-static-libgcc" here:
-  extra_config+=(
+  _extra_config+=(
     '--with-boot-ldflags="-static-libstdc++"'
     '--with-stage1-ldflags="-static-libstdc++"'
   )
@@ -284,7 +281,7 @@ build() {
     --with-bugurl="https://github.com/msys2/MINGW-packages/issues" \
     --with-gnu-as \
     --with-gnu-ld \
-    "${extra_config[@]}"
+    "${_extra_config[@]}"
 
   # While we're debugging -fopenmp problems at least.
   # .. we may as well not strip anything.

